tests/integration/schema_tests_test/models-v2/bq-models/schema.yml

```diff
@@ -12,10 +12,18 @@ models:
       - name: favorite_color
         tests:
           - accepted_values: { values: ['blue', 'green'] }
+      - name: ephemeral_copy
+        description: "Testing a column with the same name as the model"
+        tests:
+          - not_null
+          - unique
```

Comment:
@jtcohen6 I added a test to cover ([#33](https://github.com/dbt-labs/dbt-bigquery/issues/33) as well